buildSrc/src/main/groovy/org/elasticsearch/gradle/precommit/ThirdPartyAuditTask.groovy

```diff
@@ -40,17 +43,52 @@ import java.util.regex.Pattern;
  * Basic static checking to keep tabs on third party JARs
  */
 public class ThirdPartyAuditTask extends AntTask {
-    
+
     // patterns for classes to exclude, because we understand their issues
-    private String[] excludes = new String[0];
-    
+    private List<String> excludes = [];
+
+    @Input
```

Comment:
These don't need to be accessible or use the input annotations. They can be added as inputs in the ctor using the `inputs` member that all tasks have.